EndlessClient/UIControls/ScrollBar.cs

```diff
@@ -68,6 +68,7 @@ public class ScrollBar : XNAControl
             _scrollButton.SetParentControl(this);
 
             _totalHeight = DrawAreaWithParentOffset.Height;
+            this.ScrollToTop();
```

Comment:
I think the `ScrollToTop` call here is a no-op since `ScrollOffset` is already set to `0`

If you want the `+1` offset then just add that to the `_scrollButton` position parameter in its constructor call.